iocore/dns/DNS.cc

```diff
@@ -32,6 +32,8 @@
 #define SRV_SERVER (RRFIXEDSZ + 6)
 #define SRV_FIXEDSZ (RRFIXEDSZ + 6)
 
+#define RCODE_ARRAY_SIZE 11
+
```

Comment:
We don't need `RCODE_ARRAY_SIZE`. If we initialize an array without an explicit size, the compiler figures out the size for us. Thus:

```
  const char *RCODE_NAME[] = {
     "NOERROR", "FORMERR", "SERVFAIL", "NXDOMAIN", "NOTIMP", "REFUSED", "YXDOMAIN", "YXRRSET", "NXRRSET", "NOTAUTH", "NOTZONE",
   };
```

Reply:
Sounds good! (: